pkg/internal/checkup/executor/console/console.go

```diff
@@ -102,13 +110,13 @@ func SafeExpectBatchWithResponse(serialConsoleClient vmiSerialConsoleClient,
 	vmiName string,
 	expected []expect.Batcher,
 	timeout time.Duration) ([]expect.BatchRes, error) {
-	expecter, _, err := NewExpecter(serialConsoleClient, vmiNamespace, vmiName, timeout)
+	expecter := NewExpecter(serialConsoleClient, vmiNamespace, vmiName)
+	gGExpect, err := expecter.spawnConsole(timeout)
```

Comment:
What is the meaning of `gGExpect`?

Reply:
I'm guessing google expect, but I don't know - I didn't make it up :)

Reply:
but I did have a typo.. will change to `gExpect` in all commits

Reply:
fixed